.github/workflows/rebuild-old-docs.yml

```diff
@@ -0,0 +1,100 @@
+name: Rebuild old docs
```

Comment:
```suggestion
#
# This workflow lets us rebuild and redeploy an old version of the documentation.
# Running it manually, specify a version and a git commit hash to deploy to that labeled version of the documentation.
# This can be used to either publish old versions of the documentation, or to replace the current version without cutting a release.
#
name: Rebuild old docs
```